specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/Jobs.json

```
@@ -935,6 +935,64 @@
       "format": "int32",
       "type": "integer"
     },
+    "PollingInterval": {
+      "description": "Interval to check each event source in seconds. Defaults to 30s",
+      "format": "int32",
+      "type": "integer"
+    },
+    "ScaleRule": {
+      "description": "Scaling rule.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Scale Rule Name",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of the scale rule\neg: azure-servicebus, redis etc.",
+          "type": "string"
+        },
+        "metadata": {
+          "description": "Metadata properties to describe the scale rule.",
+          "type": "object"
+        },
+        "auth": {
+          "description": "Authentication secrets for the scale rule.",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/ScaleRuleAuth"
+          }
+        }
+      }
+    },
+    "Scale": {
```

Rename to fix `PreCheck/DuplicateSchema | Duplicate Schema named 'Scale'` for azure-sdk-for-go check

```suggestion
    "JobScaleConfiguration": {
```

---

This will not work because the requested json name for this is scale.